### PR TITLE
chore: add volunteer dev-divyansh

### PR DESCRIPTION
**Pull request by dev-divyansh**
Please review my PR

### CODE REVIEW

File: data/users/dev-divyansh.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Divyansh Pratap Singh",
+  "bio": "Native Android Developer",
+  "image": "https://en.everybodywiki.com/images/f/f9/IM_pratap_divyansh.jpg",
+  "categories": [
+    {
+      "slug": "Mobile-Development",
+      "topics": ["Koltin","Java","Android"]
```
Comment (codewithvoid) [inline data/users/dev-divyansh.json:8]: remove Kotlin and Java as they are already mentioned in programming.

fix casing:
"mobile-development"

File: data/users/dev-divyansh.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    },
+    {
+      "slug": "programming",
+      "topics": ["Python" , "Java"  , "Kotlin" , "Go-lang" , "Ruby" ]
```
Comment (codewithvoid) [inline data/users/dev-divyansh.json:12]: Limit the topics to 2-3

**Review by codewithvoid — CHANGES_REQUESTED** [review]

File: data/users/dev-divyansh.json (hunk 1 above)
Comment (dev-divyansh) [reply data/users/dev-divyansh.json]: updated the changes

File: data/users/dev-divyansh.json (hunk 2 above)
Comment (dev-divyansh) [reply data/users/dev-divyansh.json]: updated the changes

**Comment by dev-divyansh** [comment]
Please review it !
I had updated the changes

**Comment by codewithvoid** [comment]
Perfect. Welcome. 

Join community discord [here](https://discord.gg/DFwsFKfgEB).

**Review by codewithvoid — APPROVED** [review]

**Pull request merged**